backend/devices/bitbox02/device.go

```diff
@@ -338,9 +339,18 @@ func (device *Device) CreateBackup() error {
 	if device.status != StatusSeeded {
 		return errp.New("invalid status")
 	}
+
+	timestamp := time.Now().Unix()
```

Comment:
```
now := time.Now()
_, offset := now.Zone()
```
and inline the other vars? i'm a fan of not having single-use vars (and the naming questions fall away).